qa_forum/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import User
```

Comment:
@chuckcwh It's fine. Just want to make sure that you don't do this in a real project.